bciers/apps/coam/app/layout.tsx

```diff
@@ -0,0 +1,16 @@
+export const metadata = {
```

Comment:
Hey do you want to replace this entire file with the reusable layout that Shon recently added? Can just copy the layout from `/bciers/apps/dashboard/app/layout.tsx`.

That way this app will look like a bc gov app :)